_inc/connect-button.js

```diff
@@ -72,7 +72,7 @@ jQuery( document ).ready( function( $ ) {
 		},
 		handleAuthorizationComplete: function() {
 			jetpackConnectButton.isRegistering = false;
-			location.reload();
+			window.location.hash = '#/plans-prompt';
```

Comment:
Will this be enough once the connect button lives in multiple places in the dashboard (on the plugins page for example), and not just in `/wp-admin/admin.php?page=jetpack`?

Reply:
This has been addressed in #f853a00.